### ISSUE TITLE
I have some problem about solver.py

### ISSUE THREAD

**Issue by silencedog** [AceCoooool/DSS-pytorch#22]
What means                             self.net.base.load_state_dict(torch.load(self.config.vgg)) (in line 59)  . If you kown please tell me      THX!

**Comment by silencedog** [comment]
this vgg is pre-training? or other?

**Comment by AceCoooool** [comment]
yes, this is load the pretrained vgg features (pretrained from imagenet --- here using the pytorch pretrained model)

**Comment by silencedog** [comment]
> yes, this is load the pretrained vgg features (pretrained from imagenet --- here using the pytorch pretrained model)

THX

**Comment by silencedog** [comment]
大佬您好，您的DSS的pytorch版本 那个solver.py中的F-Measure运算是不是错的呀，beat的平方默认为0.3，而您是将beat=0.3。中间相差挺多的




| |
潘
邮箱：18752067139@163.com
|

签名由 网易邮箱大师 定制

On 03/21/2019 20:01, Ace wrote:

yes, this is load the pretrained vgg features (pretrained from imagenet --- here using the pytorch pretrained model)
